deploy/harvester-cloud-provider/values.yaml

```diff
@@ -0,0 +1,62 @@
+# Default values for harvester-cloud-provider.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 2
+
+image:
+  repository: rancher/harvester-cloud-provider
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
```

Comment:
the default tag should be `master-head` for the master branch and update the pullPolicy accordingly?